internal/broker/broker.go

```diff
@@ -681,7 +663,7 @@ func (b *Broker) fetchUserInfo(ctx context.Context, session *sessionInfo, t *aut
 	}
 
 	if userClaims.Email == "" {
-		return claims{}, nil, errors.New("invalid claims")
+		return claims{}, nil, errors.New("user email is required, but was not provided")
```

Comment:
that + the decorate function will really help next time the config on Entra ID is invalid :p